docs/pages/tabs.js

```diff
@@ -193,6 +435,20 @@ function TabExample() {
   />,
 );
 
+card(
+  <MainSection name="Related">
+    <MainSection.Subsection
+      description={`
+**[Link](/Link)**
+Link is used to navigate to different areas of the product or to external sites. Link is the preferred component in cases where you want to direct the user to unrelated content.
+
+**[SegmentedControl](/SegmentedControl)**
+SegmentedControl is used to switch between views within a small area of content, such as a [Popover](/Popover). SegmentedControl is preferred when changing state or selection within a view.
```

Comment:
I'm confused how SelectedControl is related to Popover? There could be one in a Popover, but I wouldn't have thought to associate them. It's just switching between panels of content, right?

Reply:
Yeah, the whole SegmentedControl vs. Tabs things is a bit wonky right now because are Tabs aren't tabs :P I think this was added since our most use case for SegmentedControl tends to be within a Popover